cli/http.go

```diff
@@ -182,10 +182,9 @@ func CreateHTTPRawRequest(method, urlPath, urlQuery, payload, contentType string
 		log.Fatalf("Unable to parse DC/OS Cluster URL '%s': %s", dcosUrl, err)
 	}
 	if len(dcosAuthToken) == 0 {
-		dcosAuthToken = RequiredCLIConfigValue(
-			"core.dcos_acs_token",
-			"DC/OS Authentication Token",
-			"Run 'dcos auth login' to log in to the cluster.")
+		// if the token wasnt manually provided by the user, try to fetch it from the main CLI.
+		// this value is optional: clusters can be configured to not require any auth
+		dcosAuthToken = OptionalCLIConfigValue("core.dcos_acs_token")
```

Comment:
@nickbp: Can we log a warning here?  Concerned that it will fail now on clusters with authentication enabled in an unclear way.